ajax/libs/vue-tables-2/package.json

```
@@ -0,0 +1,30 @@
+{
+  "name": "vue-tables-2",
+  "filename": "vue-tables-2.min.js",
+  "description": "Vue.js 2 grid components",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/matfish2/vue-tables-2"
+  },
+  "keywords": [
+    "vue",
+    "vuex",
+    "bootstrap",
+    "table",
+    "grid"
+  ],
+  "author": "Matanya Fishaimer <matfish2@gmail.com>",
+  "license": "MIT",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/matfish2/vue-tables-2.git",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "**/*.+(js)"
```

```suggestion
          "*.js"
```
Let's use a simpler glob expression here.